.gitignore

```diff
@@ -258,3 +258,72 @@ paket-files/
 *.log
 *.cso
 *.zip
+
+# Xcode - Swift
+# Github's gitignore for Swift: https://github.com/github/gitignore/blob/master/Swift.gitignore
```

Comment:
we can put this in a nested .gitignore, that will align better with what we've done with other clients (like [Unity](https://github.com/CatalystCode/3dtoolkit/blob/master/Samples/Client/Unity/.gitignore))

Reply:
👍